sys/include/net/netopt.h

```diff
@@ -645,6 +645,23 @@ typedef enum {
      */
     NETOPT_LORAWAN_MIN_RX_SYMBOL,
 
+    /**
+     * @brief   (uint8_t*) phy layer syncword
+     */
+    NETOPT_SYNCWORD,
```

Comment:
is this LoRa specific? Then it should be named `NETOPT_LORA_SYNCWORD` or `NETOPT_LORAWAN_SYNCWORD` or even `NETOPT_LORAWAN_PHY_SYNCWORD`?

Reply:
This is a general concept for synchronizing data transmission (see [wikipedia](https://en.wikipedia.org/wiki/Syncword)), so no this is not LoRa specific.

Reply:
yup. In most 15.4 radios it's possible to change the SFD (Syncword) to a different value